### PR TITLE
adding an EDA with recommendation = best so far.

### PR DESCRIPTION
**Pull request by teytaud**
Issue https://github.com/facebookresearch/nevergrad/issues/349 suggested this ?

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

https://github.com/facebookresearch/nevergrad/issues/349

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by teytaud** [comment]
@ARF1

File: nevergrad/optimization/optimizerlib.py [hunk 1]
```diff
@@ -437,7 +437,15 @@ def _internal_tell(self, x: ArrayLike, value: float) -> None:
             self.evaluated_population_sigma = []
             self.evaluated_population_fitness = []
 
+            
+# We have seen that NaiveTBPSA is a great extension of TBPSA for the noise-free case.
+# ARF1 suggested that something similar could happen for MPCEDA.
```
Comment (LaurentMnr95) [inline nevergrad/optimization/optimizerlib.py]: Not sure this comment is clear for everyone, change this before approving

Comment (ARF1) [reply nevergrad/optimization/optimizerlib.py]: Also, this is probably far too generous towards me. I was just complaining that the optimizer did not seem to work as I had expected - while apparently misunderstanding that the optimizer was indeed working as it should. Thanks for being so responsive though!

Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py]: @arf1: is MPCEDA or NaiveMPCEDA good for you ? If yes that's a good reason for adding it.

Comment (ARF1) [reply nevergrad/optimization/optimizerlib.py]: @teytaud I have not tried it yet but I will give it a try.

As I am not an expert in optimization I naively ran all optimizers against my problem (NN reinforcement learning) to see which came out on top. I will give NaiveMPCEDA a try. Though from what I have seen so far, I expect other algorithms are better suited.

Comment (ARF1) [reply nevergrad/optimization/optimizerlib.py]: @teytaud For me NaiveMPCEDA is better than MPCEDA but neither is really well-suited to my problem. I get the best results (absolute result as well as result vs. speed) with CMA-variants and DE-variants.

Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py]: thanks @arf1

**Pull request closed without merging**